build-pkgs/abuild/build.yml

```diff
@@ -0,0 +1,3 @@
+image: build-pkgs_abuild
```

Comment:
Any chance you can call it build-pkgs-abuild ? Or is underscore somehow meaningful?

Reply:
Hi @rvs   - the idea is that '-' acts as a separator and '_' acts as a proxy for '/'

Since the abuild package is in build-pkgs/abuild I thought that this will be consistent, while providing a sub-namespace.
zededa/build-pkgs_abuild
zededa/build-pkgs_xxx
etc. for package building related tooling.

Does that make sense ?

Reply:
Makes sense. I'm still a bit taken aback by a mixture of '-' and '_' in a single identifier, but at least I see what you meant by it. 

It allows for a certain level of namespace separation which is good. Lets go ahead with this and if we come up with a better naming scheme in the future we can always rename it.

Reply:
Ack, thanks!